mne/gui/_stc.py

```diff
@@ -0,0 +1,268 @@
+# -*- coding: utf-8 -*-
+"""Source estimate viewing graphical user interfaces (GUIs)."""
+
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import platform
+
+from scipy.ndimage import maximum_filter
+
+from qtpy import QtCore, QtGui
+from qtpy.QtCore import Slot, Signal
+from qtpy.QtWidgets import (QVBoxLayout, QHBoxLayout, QLabel,
+                            QMessageBox, QWidget, QAbstractItemView,
+                            QListView, QSlider, QPushButton,
+                            QComboBox)
+
+from matplotlib.colors import LinearSegmentedColormap
+
+from ._core import SliceBrowser
+from ..transforms import apply_trans, _get_trans, invert_transform
+from ..source_estimate import _check_stc_src
+from ..source_space import _vol_src_to_image
+from ..utils import logger, _validate_type, verbose
+from .. import pick_types
```

Comment:
I get a lot of "imported but unused" warnings from GitHub for the imports above.

Reply:
Similar to previous, this was a much rougher GUI, very close to being pushed now and it won't have this cruft.

Related to your comment:

> Not sure I understand why you added the GUI here - is it not better to have that in a separate PR and then rebase and update this once the GUI is merged?

I'd like to have the full working tutorial all in one place to plan and see things at a high-level the way I'd like them to turn out all together. It doesn't all have to be merged in one giant PR, smaller PRs can be split off to make things easier for discussion. I prefer to work this way though so that I don't lose sight of the end goal/not have the pieces (separate PRs) fit together.

Reply:
Also related to working style/preference, I implemented the `apply_*_tfr` functions to return the new STC object because I think my brain would explode if I tried to work with the list of lists of stcs. I plan on implementing the list-of-lists-of-stcs-viz.to-new-stc-class function but that's rather tricky as the actual class doesn't exist yet and it should really be done the right way if it's going to be in the public API without a warning that it's experimental like the `mne.viz.Brain` class. That would mean supporting different combinations/dimensions and testing them e.g. a list of lists where the outside list is epochs and the inside list is frequencies or the outside list is source vertices and the inside list is frequencies. There's quite a few combinations and I didn't want to distract too much from the main goal since it's just a stop-gap solution so I wanted to do it last to prioritize the GUI.